swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        super().__init__(id, condition)    
+        self.fabric = fabric
+
+    def get_category(self):
+        return "Clothing"
+    
+    
+    def __str__(self):
+        return f"An object of type Clothing with id {self.id}. It is made from {self.fabric} fabric."
```

Comment:
Notice that the first part of this return string is very similar to what you have for the dunder string in the parent class! If we could make the parent's `__str__` more generic (see Item for suggestion), then we could use it as part of this string!

```suggestion
    def __str__(self):
        return f"{super().__str__()} It is made from {self.fabric} fabric."
```